### PR TITLE
Updated to ensure we are using the same query id for federated query

### PR DESCRIPTION
**Pull request by ivakegg**
plans

### CODE REVIEW

File: warehouse/query-core/src/main/java/datawave/query/planner/FederatedQueryPlanner.java [hunk 1]
```diff
@@ -334,10 +334,14 @@ public CloseableIterable<QueryData> process(GenericQueryConfiguration genericCon
             configCopy.setBeginDate(dateRange.getLeft());
             configCopy.setEndDate(dateRange.getRight());
 
+            // we want to make sure the same query and same query id are used for tracking purposes and execution
+            configCopy.setQuery(originalConfig.getQuery());
```
Comment (apmoriarty) [inline warehouse/query-core/src/main/java/datawave/query/planner/FederatedQueryPlanner.java]: Is it okay to share the same object across two instances of the planner? I would think we should copy the Query object but maybe that's an incorrect assumption on my part.

Comment (ivakegg) [reply warehouse/query-core/src/main/java/datawave/query/planner/FederatedQueryPlanner.java]: The settings object is static at this point, so no need to have a copy

Comment (apmoriarty) [reply warehouse/query-core/src/main/java/datawave/query/planner/FederatedQueryPlanner.java]: Perfect

**Review by apmoriarty — DISMISSED** [review]

File: warehouse/query-core/src/main/java/datawave/query/planner/FederatedQueryPlanner.java (hunk 1 above)
Comment (ivakegg) [reply warehouse/query-core/src/main/java/datawave/query/planner/FederatedQueryPlanner.java]: But that being said, perhaps safety.... looking at it.

Comment (ivakegg) [reply warehouse/query-core/src/main/java/datawave/query/planner/FederatedQueryPlanner.java]: looks like the query object is actually already duplicated, so all I need to do is reset the id..... testing

**Review by apmoriarty — APPROVED** [review]

**Pull request merged**